src/platform/PlatformAppProvider/helpers.ts

```diff
@@ -6,6 +6,8 @@ export type FilterParams = {
   private?: boolean;
   version?: string;
 };
+import path from "path";
+import { readFileSync } from "fs";
```

Comment:
can't afford to import `"fs"` in live-common

Reply:
`fs` use removed in last commit -> 8c9fde486222711956b856c55509dfb4c4de1847